### PR TITLE
Added $STALK messages created by GadgetPool seatalk to NMEA converter…

### PR DESCRIPTION
**Pull request by aayaffe**
… with a little additional SeaTalk functionalities

Not all Seatalk command types were added

### CODE REVIEW

**Comment by jmwooten** [comment]
Is it possible to implement this in a similar fashion to the other proprietary messages ( Garmin, Trimble,  Ashtech, etc..)?

**Comment by aayaffe** [comment]
@jmwooten 
I thought about that but unlike the other propriety messages, this one does not start with P and that is why the  Regex picks it up as a standard NMEA message.

**Comment by jmwooten** [comment]
Got it. That's unfortunate.

File: pynmea2/seatalk_utils.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            '50': 'LAT position',
+            '51': 'LON position',
+            '52': 'Speed over Ground',
+            '53': 'Course over Ground',
+            '82': 'Target waypoint name',
+            '84': 'Compass heading  Autopilot course and Rudder position',
+            '9C': 'Compass heading and Rudder position'
+        }
+        try:
+            return byte_to_command[self.cmd]
```
Comment (Knio) [inline pynmea2/seatalk_utils.py]: replace this try/except with: `byte_to_command.get(self.cmd, 'Unknown Command`)

File: pynmea2/seatalk_utils.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# pylint: disable=invalid-name
+class SeaTalk(object):
+    '''Mixin to add Seatalk functionality. Based on Thomas knauf's work
+    http://www.thomasknauf.de/seatalk.htm'''
+
+    # pylint: disable=no-member
+    @property
+    def command_name(self):
+        '''Get seatalk command's meaning'''
+        byte_to_command = {
```
Comment (Knio) [inline pynmea2/seatalk_utils.py]: Make this a class member so it's not recreated every time

**Comment by Knio** [comment]
Thanks! +1 unfortunate that this can't be part of the proprietary messages, but nobody seems to follow the spec. I would like to refactor the parser a bit later so that it's possible, but I won't block this useful addition on that.

**Pull request merged**